### ISSUE TITLE
[ADE 508 Audit]: VA Notify Portal: 508 Audit Report for 43660 VA Notify Portal - Defect 5 - High

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#109087]
> [!NOTE]
> **QUICK SUMMARY:** Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/109082 to track the work related to Defect 5 - High

### Defect 5 | High | Created:
_WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability._

### Feedback from Project Team
A - Throughout the product, elements do not receive focus in an order that preserves meaning and operability. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Home), after navigating to Email or SMS: How to choose and pressing tab twice followed by shift + tab twice, the focus is not returned to Email or SMS: How to choose. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!Gn52wf00pm_uaWSi4pSvv-hyoT6od3XT5gCnSH0R2NHGb7X_irffpaDL7QnZwXjqcqoc9fhACWCMkX955vuHX9LzcHk$)[ANY PAGE] 
(This is the result of a known bug with our "Back to top" button. We will be removing and implementing the component. 

B- Throughout the product, some elements do not receive focus in an order that preserves meaning and operability. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Add Template -> Email), tab cannot be used to move the focus to Cancel. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!Gn52wf00pm_uaWSi4pSvv-hyoT6od3XT5gCnSH0R2NHGb7X_irffpaDL7QnZwXjqcqoc9fhACWCMkX955vuHX9LzcHk$)[ANY PAGE] 
(This is the result of a known bug with our "Back to top" button. We will be removing and implementing the component.)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
